src/checkqueue.h

```diff
@@ -132,16 +139,31 @@ class CCheckQueue
     {
     }
 
+    //! Create a pool of new worker threads.
+    void StartWorkerThreads(const int threads_num)
+    {
+        if (threads_num <= 0) return;
```

Comment:
Is this needed? the for loop will never run anyway when this is true.

Reply:
The idea was to prevent `m_stopped = false;`

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/18710#issuecomment-619008172).